packages/datatrak-web/src/Routes.tsx

```diff
@@ -102,14 +102,34 @@ const SurveyStartRedirect = () => {
 const SurveyPageRedirect = ({ children }) => {
   const { screenNumber } = useParams();
   const { visibleScreens } = useSurveyForm();
+
   if (visibleScreens && visibleScreens.length && visibleScreens.length < Number(screenNumber)) {
     return <SurveyStartRedirect />;
   }
   return children;
 };
 
+/**
+ * This is to redirect the user to the survey not found page if they try to access a survey that does not exist
+ */
+const SurveyNotFoundRedirect = ({ children }) => {
```

Comment:
This is a good idea. 
On a side note this file is getting a bit big. Maybe next time we add something we should split it up a bit